app/renderer/component/App.tsx

```diff
@@ -0,0 +1,12 @@
+var React = require('react');
+import { hot } from "react-hot-loader/root";
+import { Heading } from './components';
+
+export default hot((): JSX.Element =>
```

Comment:
hot -> App

Reply:
We can not change it, I think it is name of function or class

Reply:
> We can not change it, I think it is name of function or class

`import { export1 as alias1 } from "module-name";`
[link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)